app/Models/Article.php

```diff
@@ -115,6 +115,16 @@ public function scopeSortByPublishedDate(Builder $query): Builder
         return $query->orderBy('articles.published_at', 'desc');
     }
 
+    /**
+     * @param  Builder<self>  $query
+     * @return Builder<self>
+     */
+    public function scopePublished(Builder $query): Builder
```

Comment:
There is already a scope called `isPublished()` in this model that only checks for `published_at` being `null` and is used in a [`CollectionController`](https://github.com/ArdentHQ/dashbrd/blob/feat/articles/app/Http/Controllers/CollectionController.php#L262).

Can you just ensure that we can remove that other scope and replace the call in the controller with your scope?